chrome/content/zotero/xpcom/data/feedItem.js

```diff
@@ -249,8 +249,28 @@ Zotero.FeedItem.prototype.translate = Zotero.Promise.coroutine(function* (librar
 	translate.getTranslators();
 	let translators = yield deferred.promise;
 	if (!translators || !translators.length) {
-		Zotero.debug("No translators detected for feed item " + this.id + " with URL " + this.getField('url'), 2);
-		throw new Zotero.Error("No translators detected for feed item " + this.id + " with URL " + this.getField('url'))
+		Zotero.debug("No translators detected for feed item " + this.id + " with URL " + this.getField('url') + 
+			'\nCloning item instead', 2);
```

Comment:
Nope. Again: " -- cloning item instead"

(For textual notes, I'm almost always giving you something you can just copy and paste.)